BekkaN/assets/google.js

```diff
@@ -0,0 +1,102 @@
+// // var geocoder;
```

Comment:
It looks like this entire js file is commented out, should we just delete instead?